webproject/src/main/java/com/google/sps/dataManagers/OrderManager.java

```diff
@@ -81,6 +85,17 @@ public Order addOrder(String userEmail, String product, int quantity, double uni
         return dataSnapshots.get().stream().map(this::toOrder).collect(Collectors.toList());
     }
 
+    public Order deleteOrder(String orderId) throws ServletException {
+        if (!isValidOrder(orderId)) {
+            throw new ServletException("Invalid orderId");
+        }
+
+        Order order = getOrder(orderId);
```

Comment:
What is this line for again?

Reply:
I thought we might like to return the order that is being deleted once it is deleted. I can remove it and have the return type be void instead? Or get the Order from the delete function and return that Order instead?